These tests require that Mathics already be installed to work,
which is not true when nix runs them.

--- a/test/test_console.py	2015-09-07 21:41:08.530501979 -0700
+++ b/test/test_console.py	2015-09-07 21:42:44.082176084 -0700
@@ -13,6 +13,7 @@
         os.environ["TERM"] = "dumb"
         self.console = pexpect.spawn('python2 mathics/main.py --color NOCOLOR')
 
+    @unittest.expectedFailure
     def testLaunch(self):
         cons = self.console
 
@@ -41,6 +42,7 @@
             'Quit by pressing CONTROL-D\r\n'
             '\r\n')
 
+    @unittest.expectedFailure
     def testPrompt(self):
         cons = self.console
         cons.expect('Quit by pressing CONTROL-D\r\n\r\n')
